breeze

```diff
@@ -1015,24 +1003,6 @@ function breeze::parse_arguments() {
             # if not set here, docker cached is determined later, depending on type of image to be built
             shift
             ;;
-        -P | --force-pull-images)
```

Comment:
No need to force-pull the image or to pull base python image. We will not let the user (by default) to rebuild the image before rebasing to latest main (it can be forced but it fails by default) so pulling latest python image and latest cache should always be good.